sdk/nodejs/tests/runtime/testdata/closure-tests/cases/116-Do-not-capture-2/snapshot.txt

```diff
@@ -7,8 +7,8 @@ function __f0() {
     with({ message: "Function 'b' cannot be called at runtime. It can only be used at deployment time.\n\nFunction code:\n  () => console.log(\"the actual function\")\n", this: undefined, arguments: undefined }) {
 
 return () => {
-                throw new Error(message);
-            };
+            throw new Error(message);
+        };
```

Comment:
The indentation changed in the transpiled .js code because it's no longer using the `__awaiter` helper that TypeScript emits, it's using async/await directly.

Reply:
I am slightly surprised there's not more churn in the snapshots 🤷